### DISCUSSION TITLE
Opposite of no-extra-boolean-cast?

### DISCUSSION THREAD

**Discussion by ramarivera** [eslint/eslint#15601] [Q&A]
Hey folks!

Does anyone know if there is a rule which does pretty much the opposite of [no-extra-boolean-cast](https://eslint.org/docs/rules/no-extra-boolean-cast) ? 

We want to align in a single style in my workplace, and the usage of `if(!!somethingThatIsNotABool)` won, so trying to find out if such a rule exists.

Thanks in advanced :)

**Comment by nzakas** [comment]
Not to my knowledge, but you can always create a custom rule or use no-restricted-syntax to do this.
